equivalences.csv

```diff
@@ -1582,3 +1582,50 @@ SIGNOR,SIGNOR-PF6,AMPK_alpha
 SIGNOR,SIGNOR-PF7,p14_3_3
 SIGNOR,SIGNOR-PF8,FGF
 SIGNOR,SIGNOR-PF9,BMP_receptor
+HGNC_GROUP,161,Ligand_gated_ion_channels

```

Comment:
Why not use HGNC_GENEFAMILY to better mirror identifiers.org?

Reply:
I just followed the existing entries, I'm not against renaming them all together, just not on this PR necessarily